default/scripting/buildings/XENORESURRECTION_LAB.focs.txt

```diff
@@ -13,6 +13,26 @@ BuildingType
         ]
     ]
     EnqueueLocation = [[ENQUEUE_BUILD_ONE_PER_PLANET]]
+    effectsgroups = [
+        EffectsGroup
+            scope = And [
+                Source
+                ContainedBy HasSpecial name = "EXTINCT_BANFORO_SPECIAL"
```

Comment:
`scope = Source
activation = And [
    Object id = Source.PlanetID
    HasSpecial name = "EXTINCT_BANFORO_SPECIAL"
]`
Does not work (parses ok, does not activate).